mantle/system/ns/exec.go

```diff
@@ -33,41 +34,71 @@ func Command(ns netns.NsHandle, name string, arg ...string) *Cmd {
 }
 
 func (cmd *Cmd) CombinedOutput() ([]byte, error) {
+	var err error
 	nsExit, err := Enter(cmd.NsHandle)
 	if err != nil {
 		return nil, err
 	}
-	defer nsExit()
+	defer func() {
+		e := nsExit()
+		if err != nil {
+			err = fmt.Errorf("%v; %v", err, e)
+		} else {
+			err = e
+		}
+	}()
 
 	return cmd.ExecCmd.CombinedOutput()
```

Comment:
one more change:
```
err  = cmd.ExecCmd.CombinedOutput()
return err
```
Otherwise the deferred `func` doesn't do anything with the error.